### PR TITLE
FilePicker Tiles view render pages

### PR DESCRIPTION
**Pull request by VictorRomanov1986**
| Q               | A
| --------------- | ---
| Bug fix?        | [X ]
| New feature?    | [ ]
| New sample?      | [ ]
| Related issues?  | fixes #X, partially #Y, mentioned in #Z

#### What's in this Pull Request?

This PR fixes an issue, when List rendering more than 1 pages of items and selecting the last item led to scrolling list to the top and displaying only first page.

### CODE REVIEW

File: src/controls/filePicker/controls/TilesList/TilesList.tsx [hunk 1]
```diff
@@ -76,6 +76,8 @@ export class TilesList extends React.Component<ITilesListProps> {
               getItemCountForPage={this._getItemCountForPage}
               getPageHeight={this._getPageHeight}
               onRenderPage={(pageProps: IPageProps, defaultRender?: IRenderFunction<IPageProps>) => this._onRenderPage(pageProps, defaultRender)}
+              renderedWindowsAhead={this.props.items.length}
```
Comment (AJIXuMuK) [inline src/controls/filePicker/controls/TilesList/TilesList.tsx:79]: Isn't that too much of extra windows?
If I have 20 items - we will render extra 40 windows.

**Review by AJIXuMuK — CHANGES_REQUESTED** [review]
Thanks @VRomanovTau for the PR!
I have one question in the review - could you please answer/address it?

File: src/controls/filePicker/controls/TilesList/TilesList.tsx (hunk 1 above)
Comment (VRomanovTau) [reply src/controls/filePicker/controls/TilesList/TilesList.tsx:79]: I am not really sure what these properties are responsible for, but without having them in such values component re-renders, when selecting the last item when there is more than 2 pages and displaying only 1st page. With less value it starts to re-render and randomly scrolling to random position

**Comment by AJIXuMuK** [comment]
Sorry @VRomanovTau for the long delay. 
Do you have some test I could you to repro the issue?
I don't think the correct fix is to render so many pages.

**Comment by michaelmaillot** [comment]
Hello @VictorRomanov1986,

Did you have to the chance to post some tests to repro the issue?

**Comment by michaelmaillot** [comment]
Hello @VictorRomanov1986,

What's the status of this PR? Do you need any assistance?

**Comment by michaelmaillot** [comment]
Ok, feel free to open a new one anytime 🙂

**Pull request closed without merging**